.github/actions/release-azure-artifact/action.yml

```diff
@@ -0,0 +1,61 @@
+name: Release Azure Function Artifacts
+description: Release Azure Function Artifacts
+
+inputs:
+  plugin_version:
+    description: 'The version of the plugin'
+    required: true
+  plugin_path:
+    description: 'The path of the plugin'
+    required: true
+  plugin_name:
+    description: 'The name of the plugin'
+    required: true
+  plugin_functions:
+    description: 'The name of the plugin functions'
+    required: true
+  repo-token:
+    description: 'The token to use for the GitHub API'
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Init Python Environment
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+        cache: 'pip'
+
+    - name: Install Packages
+      shell: bash
+      run: |
+        for function in $(echo '${{ inputs.plugin_functions }}' | jq -r '.[]'); do
+          pip install --target ${{ inputs.plugin_path }}/$function/.python_packages/lib/site-packages -r ${{ inputs.plugin_path }}/$function/requirements.txt
+        done
+
+    - name: Zip Files
+      shell: bash
+      run: |
+        ZIP_DIR=${{ github.workspace }}/${{ inputs.plugin_path }}/zip
+        mkdir -p $ZIP_DIR
+
+        for function in $(echo '${{ inputs.plugin_functions }}' | jq -r '.[]'); do
+          cd ${{ inputs.plugin_path }}/$function
+          zip -rq $ZIP_DIR/$function.zip . -x "*.funcignore" "*requirements.txt"
+          cd ${{ github.workspace }}
+        done
```

Comment:
I'm thinking this steps (for packaging Azure function) might be reusable and worths an action. For example, for a PR build we can run a workflow to package the function, so that it can be referred from the template and tested. What do you think?

Reply:
I have added a new action called `packaging_python_azure_functions` that performs two tasks:
- Installing package from pip.
- Zipping into a zip files.